js/foam/apps/calc/CalcView.js

```diff
@@ -336,20 +339,20 @@ CLASS({
 
     .secondaryButtons {
       padding-left: 30px;
-      background: rgb(52, 153, 128);
+      background: #00796b;
     }
 
     .secondaryButtons .button {
-      background: rgb(52, 153, 128);
+      background: #00796b;
     }
 
     .tertiaryButtons {
       padding-left: 35px;
-      background: rgb(29, 233, 182);
+      background: #1DE9B6;
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#Capitalization

Reply:
Thank you!